media-video/smplayer/smplayer-16.4.0.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="Great Qt GUI front-end for mplayer/mpv"
+DESCRIPTION="SMPlayer intends to be a complete front-end for MPlayer, from \
+basic features like playing videos, DVDs, and VCDs to more advanced features \
+like support for MPlayer filters and more."
+HOMEPAGE="http://smplayer.sourceforge.net"
+COPYRIGHT="2006-2015 Ricardo Villalba"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sf.net/smplayer/smplayer-$portVersion.tar.bz2"
+CHECKSUM_SHA256="de4169c7e3a1d8bf6f3c9a77fa0a8c24c1f63db3f2b8a325bc6761c0a9d2d2ec"
+PATCHES="smplayer-$portVersion.patchset"
+ADDITIONAL_FILES="smplayer-$portVersion.rdef"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+
+PROVIDES="
+	smplayer$secondaryArchSuffix = $portVersion
+	app:SMPlayer = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	libqt4$secondaryArchSuffix >= 4.8
+	lib:libz$secondaryArchSuffix
+	cmd:mplayer
+	cmd:smtube$commandSuffix
+	"
+
+BUILD_REQUIRES="
+	libqt4${secondaryArchSuffix}_devel >= 4.8
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	# TODO: fix paths
+	#		DATA_PATH
+	#		DOC_PATH
+	#		TRANSLATION_PATH
+	#		THEMES_PATH
+	#		SHORTCUTS_PATH
+	echo "SVN-r6966" > svn_revision
```

Comment:
Good catch, thanks. It builds and works fine even if we skip the creation of file svn_revision.
I have therefore removed that line and updated the commit.
I did not dare to remove, however, the commented lines of the "TODO: fix paths" block.
(Should I remove them too?)